src/lib/functions/runtimes/js/index.ts

```diff
@@ -5,54 +5,36 @@ import { Worker } from 'worker_threads'
 
 import lambdaLocal, { type LambdaEvent } from 'lambda-local'
 
-import type { BuildFunction, GetBuildFunction, InvokeFunction, OnDirectoryScanFunction } from '../index.js'
+import type { BuildFunction, GetBuildFunction, InvokeFunction } from '../index.js'
 import { BLOBS_CONTEXT_VARIABLE } from '../../../blobs/blobs.js'
 import type NetlifyFunction from '../../netlify-function.js'
 
-import detectNetlifyLambdaBuilder, {
-  type NetlifyLambdaBuildResult,
-  type NetlifyLambdaBuilder,
-} from './builders/netlify-lambda.js'
 import detectZisiBuilder, { getFunctionMetadata, ZisiBuildResult } from './builders/zisi.js'
 import { SECONDS_TO_MILLISECONDS } from './constants.js'
 import type { WorkerMessage } from './worker.js'
 
 export const name = 'js'
 
-export type JsBuildResult = ZisiBuildResult | NetlifyLambdaBuildResult
+type SimpleJsBuildResult = {
+  schedule?: string
+  srcFiles: string[]
+}
+
+export type JsBuildResult = ZisiBuildResult | SimpleJsBuildResult
```

Comment:
There were actually _three_ different ways to run a JS function. When I removed one, it revealed that this union of two types just happened to work. This adds the missing one and removes the deprecated one... Hope that makes esnse.